packages/cli/src/theme.ts

```diff
@@ -328,18 +401,20 @@ function parseVariables(variables?: string): Record<string, string> {
   }, {} as Record<string, string>)
 }
 
+function determineFilePath(fileName: string): string | undefined {
+  const extensions = ['.ts', '.js', '.mjs', '.cjs']
+  const paths = extensions.map((ext) => resolve(process.cwd(), fileName + ext))
+  return getPath(paths) || fileName
+}
+
```

Comment:
I've broken this out into ins own function to attempt to re-use it on line 117 — but was unable to get it to work as intended. Ended up needing to add `.ts` to the path for the local theme in the tests.